### ISSUE TITLE
Can linux server terminal use nerd fonts?

### ISSUE THREAD

**Issue by ChandlerHarden** [ryanoasis/nerd-fonts#535]
If there is only a Linux server terminal without GUI or KDE. Can nerd fonts still work in the terminal?

**Comment by adrian5** [comment]
TL;DR: **Yes.**

Something in your pipeline does the rendering for your eyes to see. It cannot differentiate between a common letter and an exotic shape, it just renders the glyphs it's being asked to.

If your server has no graphics stack, then you're likely logging into it remotely (e.g. SSH). In that case, the server only sends characters back to your local machine, which then decides how to render them, and with which font. That's why you can change your terminal's font during a remote session without talking to the server about it.

So, as long as the server supports UTF-8, it doesn't have to understand how these characters are actually turned into readable glyphs for human consumption.

**Comment by ryanoasis** [comment]
Thanks. Yes.
